Author: pchunt <peter.hunt@opengear.com>
Date:   Tue Oct 17 10:20:22 2017 -0600

    Fix 3G registration change when sending SMS

    Dont rely on 3GPP info when modem is in 3GPP2 mode

    Fixes a problem where sending a CDMA text message can sometimes
    change the technology from LTE to 1xRTT (CDMA/EvDO)

    OG-5130
Index: ModemManager-1.14.0/src/mm-broadband-modem-qmi.c
===================================================================
--- ModemManager-1.14.0.orig/src/mm-broadband-modem-qmi.c
+++ ModemManager-1.14.0/src/mm-broadband-modem-qmi.c
@@ -2653,6 +2653,8 @@ common_process_serving_system_3gpp (MMBr
         (selected_network == QMI_NAS_NETWORK_TYPE_UNKNOWN &&
          (mm_access_technologies & MM_IFACE_MODEM_3GPP_ALL_ACCESS_TECHNOLOGIES_MASK))) {
         mm_obj_dbg (self, "processing 3GPP info...");
+    } else if (selected_network == QMI_NAS_NETWORK_TYPE_3GPP2) {
+        mm_obj_dbg (self, "3GPP2 modem may not have 3GPP info...");
     } else {
         MMModem3gppRegistrationState reg_state_3gpp;
 
